src/client/datascience/jupyter/jupyterSession.ts

```diff
@@ -78,6 +81,49 @@ export class JupyterSession extends BaseJupyterSession {
         // Made it this far, we're connected now
         this.connected = true;
     }
+    public async changeKernel(
```

Comment:
Moved from JupyterSessionBase into this (this method is specific to RemoteJupyter).
The plan is to remove this as well, as a user can only change kernels by select a kernel from kernle picker, at which point we have a KernelConnectionMetadata, hence this code path should not get hit.

Right now, i'm not focusing on removing this, its a little confusing as not related to Raw kernels, hence leaving this for now.